components/event-feed-service/pkg/grpc/grpc.go

```diff
@@ -130,7 +128,7 @@ func Spawn(c *config.EventFeed, connFactory *secureconn.Factory) error {
 	}
 	defer projectUpdateManager.Stop() // nolint: errcheck
 
-	return grpcServer.Serve(conn)
+	return newGRPCServer(connFactory, c, feedStore, purgeServer).Serve(conn)
```

Comment:
Trying to reorder when the service is started to not allow the service to allow connections until it is ready to receive requests.